app/src/main/java/com/example/finding_tory/ItemViewActivity.java

```diff
@@ -0,0 +1,116 @@
+package com.example.finding_tory;
+
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+import java.util.Locale;
+import java.util.Objects;
+
+public class ItemViewActivity extends AppCompatActivity {
+    private static final int EDIT_ITEM_REQUEST = 1; // Can be any integer unique to this activity
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_item_view);
```

Comment:
We may want to set the title of the activity, because by default it's "finding-tory".
```suggestion
        setContentView(R.layout.activity_item_view);
        setTitle("View Item");
```